pkg/component/extensions/extension/extension.go

```diff
@@ -193,10 +193,13 @@ func (e *extension) Deploy(ctx context.Context) error {
 	return flow.Parallel(fns...)(ctx)
 }
 
+// WaitUntilExtensionObjectReady is an alias for extensions.WaitUntilExtensionObjectReady. Exposed for tests.
+var WaitUntilExtensionObjectReady = extensions.WaitUntilExtensionObjectReady
```

Comment:
I can't seem to find where this is being used for testing

Reply:
It's used [here](https://github.com/gardener/gardener/blob/0290b6d8ed687b9b1678c42302a64baacf2589fe/test/integration/gardenlet/seed/seed/seed_test.go#L403).